src/main/java/com/minecolonies/core/entity/ai/workers/production/EntityAIQuarrier.java

```diff
@@ -474,12 +474,12 @@ public boolean requestMaterials()
 
                 if (requestProgress.getY() != -1 && result.getIteratorPos().getY() < requestProgress.getY())
                 {
-                    requestProgress = new BlockPos(0, requestProgress.getY() + 1, 0);
+                    requestProgress = new BlockPos(-1, requestProgress.getY() + 1, 0);
```

Comment:
I noticed during stepping through the code that this caused the block in the north west corner of the layer to get skipped whilst calculating decoration blocks resources (and it looks like it is also skipped during building the quarry). I changed that as well while I was at it